specification/network/resource-manager/Microsoft.Network/2017-09-01/applicationSecurityGroup.json

```diff
@@ -0,0 +1,370 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2017-09-01"
```

Comment:
Will this be a preview API version? I thought this will not be GA API.

Reply:
It should be GA to my knowledge.

Reply:
This API Version is already rolled out on the server-side (but isn't being used by anything yet; ASGs will be the first thing to use 2017-09-01). I'm not sure what you mean by "this will not be GA API"